database/DatabaseReference/swift/ViewController.swift

```diff
@@ -101,6 +103,24 @@ class ViewController: UIViewController {
     // [END clock_skew]
   }
 
+  func writeNewUser(_ user: Firebase.User, withUsername username: String) {
+    // [START rtdb_write_new_user]
+    self.ref.child("users").child(user.uid).setValue(["username": username])
+    // [END rtdb_write_new_user]
+  }
+
+  func writeNewUserWithCompletion(_ user: Firebase.User, withUsername username: String) {
+    // [START rtdb_write_new_user_completion]
+    self.ref.child("users").child(user.uid).setValue(["username": username], withCompletionBlock: {
+      (error:Error?, ref:DatabaseReference) in
+      if (error != nil) {
+        print("Data could not be saved.")
```

Comment:
nit: our snippets should print errors by default so when they're copy-pasted and break for some reason they produce informative errors.

```swift
if let error = error {
  print("Data could not be saved: \(error)")
}
```